packages/nodes-base/credentials/FacebookGraphApi.credentials.ts

```diff
@@ -10,7 +10,7 @@ export class FacebookGraphApi implements ICredentialType {
 
 	displayName = 'Facebook Graph API';
 
-	documentationUrl = 'facebookGraph';
+	documentationUrl = 'facebookgraph';
```

Comment:
I'm checking why this was not caught by the script

Reply:
:pray: